napi_from_js.md

```diff
@@ -0,0 +1,204 @@
+# N-API Function APIs 
+
+N-API provides a set of APIs that allow JavaScript code to 
+call back into native code. N-API APIs that support calling back
+into JavaScript take in callback function represented by 
+the `napi_callback` type. When the JavaScript VM calls back to
+native code, the `napi_callback` provided is invoked. The APIs 
+documented in this section allow the callback function to do the 
+following:
+- Get information about the context in which the callback was invoked
+- Get the arguments passed into the callback
+- Return a `napi_value` back from the callback
+
+Additionally, N-API provides a set of functions which allow calling 
+JavaScript functions from native code. One can either call a function
+like a regular JavaScript function call, or as a constructor 
+function
+
+## Functions
+
+### *napi_call_function*
+
+#### Signature
+```
```

Comment:
I forgot about this until now... should we add a 'C' after ``` to get syntax highlighting?

For example:
```C
void example(int x)
```

The [Node.js style guide](https://github.com/nodejs/node/blob/master/doc/STYLE_GUIDE.md) does suggest using language-aware code fences in documentation.